WebRoot/js/zimbraMail/mail/view/prefs/ZmAccountsPage.js

```diff
@@ -3212,6 +3212,7 @@ function() {
     } else {
      this._sendAs.focus();
     }
+	this.getButton(DwtDialog.OK_BUTTON).setText(ZmMsg.save);
```

Comment:
About indentation, this function has mix of tabs and spaces both, strange !
Fixed setting text  https://github.com/Zimbra/zm-web-client/commit/51baf34a4d64bdf1793742ac723637ba7c84bb66